core/chainio/avs_writer.go

```diff
@@ -2,15 +2,26 @@ package chainio
 
 import (
 	"context"
+	"fmt"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/avsregistry"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
 	"github.com/Layr-Labs/eigensdk-go/logging"
 	"github.com/Layr-Labs/eigensdk-go/signer"
+	gethcommon "github.com/ethereum/go-ethereum/common"
 	gethtypes "github.com/ethereum/go-ethereum/core/types"
 	servicemanager "github.com/yetanotherco/aligned_layer/contracts/bindings/AlignedLayerServiceManager"
 	"github.com/yetanotherco/aligned_layer/core/config"
 	"github.com/yetanotherco/aligned_layer/core/utils"
+	"math/big"
+	"time"
+)
+
+const (
+	maxRetries                  = 25
+	sleepTime                   = 5 * time.Second
+	incrementPercentage         = 25
+	incrementPercentageInterval = 20 * time.Second
```

Comment:
Consts should be named with caps (e.g MaxRetries, SleepTime, etc)

Reply:
Solved in [2923565](https://github.com/yetanotherco/aligned_layer/pull/330/commits/2923565d0f9c1ee104e9d3391790d6929bf653e8).